lambda_functions/source/register-organization/lambda.py

```diff
@@ -87,6 +87,38 @@ def get_active_regions(AWS_REGION):
         return my_regions
     except Exception as e:
         return e
+
+def get_ssm_regions():
```

Comment:
Recommend avoiding hard-coding region-names? consider something like to below to pick the active regions in an Account:

```
opt_status=['opt-in-not-required', 'opted-in']
filteron={'Name': 'opt-in-status', 'Values': opt_status}
regions=EC2.describe_regions(Filters=[filteron])['Regions']
region_list=[]
for region in regions: region_list.append(region['RegionName'])
```

**Output :**
```
region_list
['ap-south-1', 'eu-north-1', 'eu-west-3', 'eu-west-2', 'eu-west-1', 'ap-northeast-3', 'ap-northeast-2', 'ap-northeast-1', 'ca-central-1', 'sa-east-1', 'ap-southeast-1', 'ap-southeast-2', 'eu-central-1', 'us-east-1', 'us-east-2', 'us-west-1', 'us-w
est-2']
```

Reply:
The regions are hard coded because those are the only regions our ssm distributor supports, I don't see a way to generate this list programmatically.  This is why I created a separate function so while the other stacksets may deploy to all active regions minus Excluded regions parameter, the SSM Association stackset will only deploy to supported regions minus Excluded regions parameter.